apps/web/src/app/settings/page.tsx

```diff
@@ -1,18 +1,104 @@
 "use client";
 
 import React from "react";
-import { Typography, Box } from "@mui/material";
+import {
+  Box,
+  Typography,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  Divider,
+} from "@mui/material";
+import AccountCircleIcon from "@mui/icons-material/Person";
+import NotificationsIcon from "@mui/icons-material/NotificationsNoneOutlined";
+import PrivacyTipIcon from "@mui/icons-material/PrivacyTipOutlined";
+import GavelIcon from "@mui/icons-material/ClassOutlined";
+import SupportIcon from "@mui/icons-material/ContactSupportOutlined";
+import InfoIcon from "@mui/icons-material/InfoOutlined";
+import FeedbackIcon from "@mui/icons-material/CommentOutlined";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import { useRouter } from "next/navigation";
 
 export default function Settings() {
+  const router = useRouter();
+
+  const settingsItems = [
+    {
+      icon: <AccountCircleIcon />,
+      text: "Account",
+      route: "/settings/account",
+    },
+    {
+      icon: <NotificationsIcon />,
+      text: "Notifications",
+      route: "/settings/notifications",
+    },
+    {
+      icon: <PrivacyTipIcon />,
+      text: "Data and Privacy",
+      route: "/settings/data-privacy",
+    },
+    { icon: <GavelIcon />, text: "Legal", route: "/settings/legal" },
+    { icon: <SupportIcon />, text: "Support", route: "/settings/support" },
+    { icon: <InfoIcon />, text: "About", route: "/settings/about" },
+    {
+      icon: <FeedbackIcon />,
+      text: "Send Feedback",
+      secondaryText: "Version 1.0",
+      route: "/settings/feedback",
+    },
+  ];
+
   return (
     <Box
       sx={{
-        display: "flex",
-        justifyContent: "center",
-        alignItems: "center",
-        height: "100vh",
+        padding: "16px",
+        maxWidth: "600px",
+        margin: "0 auto",
       }}>
-      <Typography variant="h4">Settings Page</Typography>
+      <Typography variant="h4" gutterBottom>
+        Settings
+      </Typography>
+      <List>
+        {settingsItems.map(item => (
+          <React.Fragment key={item.route}>
+            <ListItem disablePadding sx={{ backgroundColor: "white" }}>
+              <ListItemButton onClick={() => router.push(item.route)}>
+                <ListItemIcon>{item.icon}</ListItemIcon>
+                <ListItemText
+                  primary={item.text}
+                  secondary={item.secondaryText || null}
+                  primaryTypographyProps={{
+                    sx: {
+                      fontWeight: item.secondaryText ? "normal" : "normal",
```

Comment:
Suggested Change:
`fontWeight: item.secondaryText ? "normal" : "inherit"`
Reason: The current setting makes no distinction; using 'inherit' allows for proper weight variation.